content/posts/blog/2023-07-07-summary-of-upcoming-changes-in-oci-image-and-distribution-specs-v-1-1.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Summary of Upcoming Changes in OCI Image and Distribution Specs v1.1
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2023-07-07
+---
+
+As mentioned in an [earlier post](./2023-01-26-oci-in-2023.md), the
+OCI Reference Types Working Group delivered a suggested set changes
+in OCI Image and Distribution specs to enable the creation and
+discovery of relationships between objects stored in an OCI registry.
+
+These changes (with some subsequent modifications) will be included in
+upcoming minor releases of both OCI Image and Distribution spec.
+
+# Notes on Versioning and the Term "OCI 1.1"
+
+The version of the upcoming minor releases of both OCI Image and
+Distribution spec happens to be the same: `v1.1.0`.
+
+This has undoubtedly led to the term "OCI 1.1" which is a bit vague and
+incorrect since these changes do not affect the Runtime spec. These versions
+of the individual specifications are all technically independent of each other.
+
+However, when dealing with clients responsible with pushing/pulling from
+registries, or registries themselves, hearing claims of "support for OCI 1.1"
+can be assumed to be relating to reference types and the changes
+outlined in this post.
+
+# List of Major Changes (3)
+
+## 1. New Registry API Endpoint for Querying Relationships
```

Comment:
IMO, we should swap this ordering:
1. guidance for generic artifacts
2. means of specifying relationships between objects (images, artifacts, and indexes alike)
3. API for querying those relationships

I realize the API is what a lot of folks are here for, but I think it's important to set the context for why the new API exists and what it's providing before we introduce it (since the target of this post is probably folks who _haven't_ been following along, not those of us who are now too familiar).

Reply:
I have no problem switching the ordering. Are you suggesting those as new headers too? Or do the current ones work?

Reply:
The current ones don't bother me too much, just the ordering (as noted by the fact that you then have to say "refer back to item 1" in both the second and third section). :sweat_smile:

If I wanted to nitpick I might reword "Official Guidance on How to Specify Alternatative Artifacts" as something like "Official Guidance on How to Create and Store Alternative (Even Non-Container) Artifacts" (including the typo fix for "Alternatative" that I didn't notice until typing this comment :smile:), but it's getting a bit wordy and perhaps should be shorter. :see_no_evil: :heart:

Reply:
done!